src/ansys/fluent/core/session.py

```diff
@@ -329,6 +339,16 @@ def download(self, file_name: str, local_directory: Optional[str] = "."):
         local_directory : str, optional
             Local destination directory. The default is the current working directory.
         """
+        warnings.warn(
```

Comment:
@hpohekar You have copied an unintended method name on the first line of the warning. You could factor out the warning call into a private method and pass in the name of the method.

Reply:
Right, Thank you.

Reply:
Done. Thank you.